src/actions/getSmartObject/GetSmartObjectAction.ts

```diff
@@ -0,0 +1,23 @@
+import Param from "../../parameters/Param";
+import Action from "../Action";
+import ParamValue from "../../parameters/ParamValue";
+
+/**
+ * @description  Extracts the original content of an embedded object of a Photoshop image.
```

Comment:
Do we need memberOf Actions -> namespace GetSmartObject?

Reply:
yup updated